fastfood/food.py

```diff
@@ -42,33 +41,23 @@ def build_cookbook(build_config, templatepack, cookbooks_home,
     with open(build_config) as cfg:
         cfg = json.load(cfg)
 
+    cookbook_name = cfg['name']
     cookbook = None
-    if cookbook_path:
-        try:
-            cookbook = book.CookBook(cookbook_path)
```

Comment:
I'll strip out the extra `cookbook = None` and the `if not cookbook`. You're right that they're unnecessary. You definitely can run `build` on an existing cookbook still though. Every method in `book` happens lazily now, so they won't blow up as long as the stencil writes out the necessary files before anyone calls any interesting methods on the cookbook.